src/bundle/Resources/public/js/scripts/core/date.time.picker.js

```diff
@@ -17,7 +17,6 @@ class DateTimePicker {
         this.inputField = this.fieldWrapper.querySelector('.ibexa-date-time-picker__input');
         this.actionsWrapper = this.fieldWrapper.querySelector('.ibexa-input-text-wrapper__actions');
         this.calendarBtn = this.actionsWrapper.querySelector('.ibexa-input-text-wrapper__action-btn--calendar');
-        this.clearBtn = this.fieldWrapper.querySelector('.ibexa-input-text-wrapper__action-btn--clear');
```

Comment:
I think we should add a listener to this button and not remove it because in HTML this button is rendered
<img width="934" alt="Screenshot 2024-07-26 at 14 39 16" src="https://github.com/user-attachments/assets/c0d4f92f-bc0c-4115-878b-45e24b3ec6e7">

Reply:
This is some remnant from version 4.2.2 where we added some classes to this button. "x" Still works without it

Reply:
https://github.com/ibexa/admin-ui/commit/0767574e78247026676a0965eb1e540ea6b7bbc9#diff-be5aa72aa4484ada67706246105a91a3ba34b4b092e29ac6aaca53219aa1ffb2